src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Json/GraphQLNamedQueryRequestJsonConverter.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using GraphQL;
+
+namespace OrchardCore.Apis.GraphQL.Json;
+public class GraphQLNamedQueryRequestJsonConverter : JsonConverter<GraphQLNamedQueryRequest>
+{
+    public static GraphQLNamedQueryRequestJsonConverter Instance = new();
```

Comment:
readonly. and add a new line at the end.
```suggestion
    public static readonly GraphQLNamedQueryRequestJsonConverter Instance = new();
    
```